build_tools/circle/build_doc.sh

```diff
@@ -204,5 +238,12 @@ then
     echo "$warnings" | sed 's/\/home\/circleci\/project\//<li>/g'
     echo '</ul></body></html>'
     ) > 'doc/_build/html/stable/_changed.html'
+
+    if [ $check ]
```

Comment:
I think you can do this without introducing the `check` variable:

```bash
if [ -n "$warnings" ]
then
    echo "There are Sphinx Warnings in the documentation!"
    echo "Please check doc/_build/html/stable/_changed.html"
    exit 1
fi
```

Reply:
I've tried: but I was using grep in order to do this and grep throws an exit 1 when it could find anything, so the script exited due to the `set -e`... I can dig deeper if it's reeeally necessary...